website/content/docs/k8s/connect/health.mdx

```diff
@@ -6,33 +6,21 @@ description: Configuring Kubernetes Health Checks
 
 # Kubernetes Health Checks in Consul on Kubernetes
 
--> 0.26+: This feature is available in consul-helm versions 0.26 and higher and is defaulted on.
-To disable it, set `connectInject.healthChecks.enabled: false`.
-
--> **Note**: As of consul-k8s `v0.26.0-beta1` and Consul Helm `v0.32.0-beta1`, syncing of the Kubernetes readiness status
-is always on, and turning it off is no longer configurable.
-
 ~> This topic requires familiarity with [Kubernetes Health Checks](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/).
 
-This page describes how to enable Consul on Kubernetes to sync the Kubernetes readiness status to Consul for service mesh uses cases.
+This page describes how Consul on Kubernetes will sync the Kubernetes health probes of a pod to Consul for service mesh use cases.
+Health check synchronization with Consul is done automatically whenever `connectInject.enabled` is `true`.
 
-When deploying your Helm chart, you can provide Helm with a custom yaml file that contains your environment configuration.
-Enabling health checks is done via the `healthChecks` stanza under `connectInject`.
+For each Kubernetes pod that is connect-injected the following will be configured:
 
-The minimal configuration required to enable health check synchronization with Consul for service mesh traffic is:
+1. A [TTL health check](/docs/discovery/checks#ttl) is registered within Consul.
+The Consul health check's state will reflect the pod's readiness status,
+which is the combination of all Kubernetes probes registered with the pod.
 
-```yaml
-global:
-  name: consul
-connectInject:
-  enabled: true
-  healthChecks:
-    enabled: true
-```
+1. If the pod is utilizing [Transparent Proxy](/docs/connect/transparent-proxy) mode the mutating webhook will also mutate all `http` based Startup, Liveness, and Readiness probes in the pod to redirect through the Envoy proxy.
+Consul will also use an `ExposePath` exception for each probe so that kubelet can access the endpoint through the Envoy proxy.
```

Comment:
```suggestion
This is done with [`ExposePaths` configuration](https://www.consul.io/docs/connect/registration/service-registration#expose-paths-configuration-reference) for each probe so that kubelet can access the endpoint through the Envoy proxy.
```

Reply:
good suggestion, just use a relative link `docs/connect/registration/service-registration#expose-paths-configuration-reference`

Reply:
Ooops! good catch, I've updated my suggestion.